language/v2/language_entities_gcs.py

```diff
@@ -0,0 +1,115 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "language_entities_gcs")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-language
+
+# sample-metadata
+#   title: Analyzing Entities (GCS)
+#   description: Analyzing Entities in text file stored in Cloud Storage
+#   usage: python3 samples/v2/language_entities_gcs.py [--gcs_content_uri "gs://cloud-samples-data/language/entity.txt"]
+
+# [START language_entities_gcs]
+from google.cloud import language_v2
+
+
+def sample_analyze_entities(gcs_content_uri):
+    """
+    Analyzing Entities in text file stored in Cloud Storage
+
+    Args:
+      gcs_content_uri Google Cloud Storage URI where the file content is located.
+      e.g. gs://[Your Bucket]/[Path to File]
+    """
+
+    client = language_v2.LanguageServiceClient()
+
+    # gcs_content_uri = 'gs://cloud-samples-data/language/entity.txt'
+
+    # Available types: PLAIN_TEXT, HTML
+    type_ = language_v2.Document.Type.PLAIN_TEXT
+
+    # Optional. If not specified, the language is automatically detected.
+    # For list of supported languages:
+    # https://cloud.google.com/natural-language/docs/languages
+    language_code = "en"
+    document = {
+        "gcs_content_uri": gcs_content_uri,
+        "type_": type_,
+        "language_code": language_code,
+    }
+
+    # Available values: NONE, UTF8, UTF16, UTF32
```

Comment:
Is there a link you can provide, so this doesn't become stale? The one you provided above for language is great!

Reply:
Added link.